orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/tasks/MonitorFront50Task.java

```diff
@@ -60,21 +65,35 @@ public TaskResult execute(@Nonnull Stage stage) {
       throw new UnsupportedOperationException("Front50 was not enabled. Fix this by setting front50.enabled: true");
     }
 
+    if (successThreshold == 0) {
+      return TaskResult.SUCCEEDED;
```

Comment:
I believe there's an issue with azure (pointed out in OSS) wherein this new task was failing ... I believe it was because there was no `lastModifiedTime` being returned.

To preserve previous behavior, the default has been switched to off.